tests/_utils-tests/model.js

```diff
@@ -12,6 +12,7 @@ import Element from '/ckeditor5/engine/model/element.js';
 import Text from '/ckeditor5/engine/model/text.js';
 import Range from '/ckeditor5/engine/model/range.js';
 import Position from '/ckeditor5/engine/model/position.js';
+import CKEditorError from '/ckeditor5/utils/ckeditorerror.js';
```

Comment:
We should use regular `Error` instances to throw inside testing tools (look at other `throws` inside this file). `CKEditorError` is used to throw errors inside the project.